### PR TITLE
Document sushi-ignoreWarnings.txt

### PR DESCRIPTION
**Pull request by ngfreiter**
Documents https://github.com/FHIR/sushi/pull/894.

⚠️ 

Do not merge until https://github.com/FHIR/sushi/pull/894 is released (And this is not a warning you should ignore).

⚠️

### CODE REVIEW

**Review by jafeltra — COMMENTED** [review]
Looks good! I had one minor wording comment.

File: content/docs/SUSHI/project/_index.md [hunk 1]
```diff
@@ -21,6 +21,20 @@ The **sushi-config.yaml** file provides project configuration data to SUSHI. It
...
 
+### Ignoring Warnings
+
+SUSHI may log warnings based on the content of the project, and these warnings can be ignored with an optional **sushi-ignoreWarnings.txt** file. Note that the contents of this file can only be used to ignore warnings. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., **simple-project/sushi-ignoreWarnings.txt** in the example above), or within the **input** directory (e.g., **simple-project/input/sushi-ignoreWarnings.txt**). Each line in the **sushi-ignoreWarnings.txt** file specifies warnings to ignore.  Warnings will be ignored if they completely match the contents of any line in the **sushi-ignoreWarnings.txt** file. The warning to ignore must be specified on only one line, and the match is case-sensitive. Additionally, regular expressions can be specified in the **sushi-ignoreWarnings.txt** file. Each regular expression must be on only one line, and a regular expression is indicated by starting and ending the line with `/`. For example, if the **sushi-ignoreWarnings.txt** file was as follows:
```
[the remark this answers is not in the document]
Comment (ngfreiter) [reply content/docs/SUSHI/project/_index.md]: Done in https://github.com/FSHSchool/site/pull/45/commits/f085163f8584d5dd24d1b96fb5f2d84cd84fdcb6

**Review by cmoesel — APPROVED** [review]
👍

File: content/docs/SUSHI/project/_index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Each FSH file can contain multiple FSH definitions of varying types. FSH file na
...
+SUSHI may log warnings based on the content of the project, which can be ignored with an optional sushi-ignoreWarnings.txt file. Note that the contents of this file can only be used to ignore warnings. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., **simple-project/sushi-ignoreWarnings.txt** in the example above), or within the **input** directory (e.g., **simple-project/input/sushi-ignoreWarnings.txt**). Each line in the **sushi-ignoreWarnings.txt** file specifies warnings to ignore.  Warnings will be ignored if they completely match the contents of any line in the **sushi-ignoreWarnings.txt** file. The warning to ignore must be specified on only one line, and the match is case-sensitive. Additionally, regular expressions can be specified in the **sushi-ignoreWarnings.txt** file. Each regular expression must be on only one line, and a regular expression is indicated by starting and ending the line with `/`. For example, if the **sushi-ignoreWarnings.txt** file was as follows:
```
Comment (jafeltra) [inline content/docs/SUSHI/project/_index.md:26]: Oh no this is completely my fault. But I think we should still bold the `sushi-ignoreWarnings.txt` reference in the first sentence.

> which can be ignored with an optional **sushi-ignoreWarnings.txt** file

**Review by jafeltra — COMMENTED** [review]
I'm sorry I have one more really small thing.

File: content/docs/SUSHI/project/_index.md (hunk 2 above)
Comment (ngfreiter) [reply content/docs/SUSHI/project/_index.md]: Oh la la ah oui oui. Fixed in 04872e1.

**Review by jafeltra — APPROVED** [review]
Spectacular!

⚠️ Reminder - do not merge this until a SUSHI version is ready ⚠️

**Pull request merged**